pkg/cmd/browse/browse.go

```diff
@@ -59,7 +59,7 @@ func NewCmdBrowse(f *cmdutil.Factory, runF func(*BrowseOptions) error) *cobra.Co
 	cmd := &cobra.Command{
 		Long:  "Open the GitHub repository in the web browser.",
 		Short: "Open the repository in the browser",
-		Use:   "browse [<number> | <path>]",
+		Use:   "browse [<number> | <commit SHA> | <path>]",
```

Comment:
To improve readability, we try to avoid spaces in `<...>` placeholders. Also, let's reorder the placeholders so that they are in order of most likely to be used first:
```suggestion
		Use:   "browse [<number> | <path> | <commit-SHA>]",
```

Reply:
Thank you for review. I got it.